### PR TITLE
Tw 1650 part i/find memory leak in chat

### PR DESCRIPTION
**Pull request by sherlockvn**
### Before:

https://github.com/linagora/twake-on-matrix/assets/43041967/9c460e3e-e6b1-4ad7-8503-aceae1aea0ad

### After:

https://github.com/linagora/twake-on-matrix/assets/43041967/eaf2400b-dc96-483b-9db7-812272a2054b

### CODE REVIEW

File: lib/widgets/matrix.dart [hunk 1]
```diff
@@ -390,7 +390,7 @@ class MatrixState extends State<Matrix>
     LoginState state,
     Client currentClient,
   ) async {
-    _cancelSubs(currentClient.clientName);
+    await _cancelSubs(currentClient.clientName);
```
Comment (hoangdat) [inline lib/widgets/matrix.dart:393]: please demo this case

File: lib/widgets/video_player.dart [hunk 2]
```diff
@@ -19,7 +19,7 @@ class VideoPlayer extends StatefulWidget {
 }
 
 class _VideoPlayerState extends State<VideoPlayer> {
-  final VideoController videoController = VideoController(Player());
+  final videoController = VideoController(Player());
```
[the remark this answers is not in the document]
Comment (sherlockvn) [reply lib/widgets/video_player.dart:22]: https://github.com/linagora/twake-on-matrix/assets/43041967/588f8fdd-9e03-46fd-be3d-56e3d7a9e2c3

**Review by Te-Z — APPROVED** [review]

**Comment by hoangdat** [comment]
- [ ] add ticket-id to ADR
- [ ] demo case: playing video after

File: lib/widgets/video_player.dart (hunk 2 above)
[the remark this answers is not in the document]
Comment (sherlockvn) [reply lib/widgets/video_player.dart:22]: There are still some leakings, i ask the owner of the packages, but its not critical right now, so we will wait for them and see. 

https://github.com/linagora/twake-on-matrix/assets/43041967/de628619-6b07-40b6-9856-91853e8ef7e8

[the remark this answers is not in the document]
Comment (sherlockvn) [reply lib/widgets/video_player.dart:22]: i also note it in ADR

**Review by hoangdat — APPROVED** [review]

**Pull request merged**